frontend/frontend-sdk/src/lib/Session.ts

```diff
@@ -2,20 +2,34 @@ import { SessionDetail } from "./events/CustomEvents";
 import { SessionState } from "./state/session/SessionState";
 import { Cookie } from "./Cookie";
 
+/**
+ * Options for Session
+ *
+ * @category SDK
+ * @subcategory Session
```

Comment:
```suggestion
 * @subcategory Internal
```
Otherwise it would be the only option listed under a new section "INTERFACES / SESSION".